jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -138,6 +140,13 @@ public ImagesAndRegistryClient call()
         }
       }
 
+      Optional<ImagesAndRegistryClient> mirrorPull =
+          tryMirrors(buildContext, progressDispatcher.newChildProducer());
+      if (mirrorPull.isPresent()) {
```

Comment:
could this message go in tryMirrors? with the name of the mirror that was successful?

Reply:
Done. Also checking the message in one of the tests.